_posts/2020-06-29-one-click-install.md

```diff
@@ -12,7 +12,7 @@ To solve this problem, the Scala Center contracted Alexandre Archambault in Janu
 For example, on Linux, all we now need is:
 
 ```bash
-$ curl -Lo cs https://git.io/coursier-cli-linux && chmod +x cs && ./cs setup
+$ curl -fL "$(curl -w "%{url_effective}\n" -I -L -s -S https://git.io/coursier-cli-"$(uname | tr LD ld)" -o /dev/null).gz" | gzip -d > cs && chmod +x cs && ./cs setup 
```

Comment:
Should I add ` && rm cs` at the end since `cs setup` installs `cs`.

Reply:
Oh! I didn't know that. Thanks! I shouldn't have spent time on figuring out `curl -w "%{url_effective}\n" -I -L -s -S https://git.io/coursier-cli-"$(uname | tr LD ld)" -o /dev/null` then. 😂